resources/production/chainspec.toml

```diff
@@ -94,6 +94,16 @@ maximum_round_length = '132seconds'
 # Expressed as a fraction (1/5 by default).
 reduced_reward_multiplier = [1, 5]
 
+[transactions]
```

Comment:
I think we'll need to keep two separate sections for the lifespan of the v2 node, until we remove support for `Deploy`s in 3.0.  I expected the new `Transaction` type would have a different set of configuration options since many configurable aspects don't have equivalents for `Deploy`.